server/models/link.ts

```diff
@@ -24,7 +24,7 @@ export async function createLinkTable(knex: Knex) {
         .inTable("domains");
       table.string("password");
       table.dateTime("expire_in");
-      table.string("target", 5000).notNullable();
+      table.string("target").notNullable();
```

Comment:
will this require a db migration?

Reply:
I mention in the description that i'm looking into this.

Reply:
There's a thread [here](https://bugsnag.slack.com/archives/C021QN6V0JW/p1690203136999059)